### PR TITLE
2847467 - Test more Quick Edit integrations.

### PR DESCRIPTION
**Pull request by balsama**
See https://www.drupal.org/node/2847467

### CODE REVIEW

File: modules/lightning_features/lightning_layout/lightning_layout.install [hunk 1]
```diff
@@ -142,6 +142,15 @@ function lightning_layout_update_8007() {
   \Drupal::service('module_installer')->install(['entity_block']);
 }
 
+
+/**
+ * Installs the Panelizer Quickedit module which allows fields placed with IPE
```
Comment (phenaproxima) [inline modules/lightning_features/lightning_layout/lightning_layout.install:147]: Nit: Needs a comma following "module".

File: modules/lightning_features/lightning_layout/lightning_layout.install [hunk 2]
```diff
@@ -142,6 +142,15 @@ function lightning_layout_update_8007() {
...
+ * to be edited with quickedit.
```
Comment (phenaproxima) [inline modules/lightning_features/lightning_layout/lightning_layout.install:148]: Quick Edit's proper name is Quick Edit, IIRC. :)

**Review by phenaproxima — COMMENTED** [review]
All I see is nitpicks. Otherwise, ready for merge.

**Comment by balsama** [comment]
merged in 5427f6b960e781e2a15131353104027f7746a310

**Pull request closed without merging**